### PR TITLE
SRCH-5350 Vote.gov link backend setup

### PR DESCRIPTION
**Pull request by krbhavith**
## Summary
- Add `show_vote_org_link` to affiliates in admin section
 
### Checklist
Please ensure you have addressed all concerns below before marking a PR "ready for review" or before requesting a re-review. If you cannot complete an item below, replace the checkbox with the ⚠️ `:warning:` emoji and explain why the step was not completed.
 
#### Functionality Checks
 
- [x] You have merged the latest changes from the target branch (usually `main`) into your branch.
  
- [x] Your primary commit message is of the format **SRCH-#### \<description\>** matching the associated Jira ticket.

- [x] PR title is either of the format **SRCH-#### \<description\>** matching the associated Jira ticket (i.e. "SRCH-123 implement feature X"), or **Release - SRCH-####, SRCH-####, SRCH-####** matching the Jira ticket numbers in the release.
 
- [ ⚠️ ] Automated checks pass. If Code Climate checks do not pass, explain reason for failures: A flakey rspec is failing, but it already passed here in the workflow before https://app.circleci.com/pipelines/github/krbhavith/search-gov/673/workflows/549059c7-856f-4ceb-a2d2-0d3c4a8b8489
 
#### Process Checks

- [x] You have specified at least one "Reviewer".

### CODE REVIEW

File: app/helpers/react_helper.rb [hunk 1]
```diff
@@ -37,6 +37,7 @@ def page_data(affiliate)
     {
       affiliate: affiliate.name,
       display_logo_only: affiliate.display_logo_only,
+      showVoteOrgLink: affiliate.show_vote_org_link && Rails.application.secrets[:enable_vote_org_link],
```
Comment (stevenbarragan) [inline app/helpers/react_helper.rb:40]: We're moving away from secrets. As it says in the ticket, we need to use the `ENABLE_VOTE_ORG_LINK` environment variable instead.

**Review by stevenbarragan — CHANGES_REQUESTED** [review]
Plase use environment variables for settings instead of secrets form now on.

**Review by stevenbarragan — CHANGES_REQUESTED** [review]
Please remove the dotenv changes but keep using an environment variable. Dotenv will be setup slightly different.

**Comment by krbhavith** [comment]
> Please remove the dotenv changes but keep using an environment variable. Dotenv will be setup slightly different.

@stevenbarragan reverted dotenv gem changes and updated with environment variable

**Review by stevenbarragan — APPROVED** [review]

**Pull request merged**